### PR TITLE
Change jetpack monthly a/b test name

### PR DESCRIPTION
**Pull request by johnHackworth**
An A/B test changed its name and date, and these changes should fix e2e tests

### CODE REVIEW

**Comment by jsnajdr** [comment]
Shouldn't this be in the Calypso repo, as the e2e tests were moved there a few days ago?

**Comment by Stojdza** [comment]
Correct! @johnHackworth Now you can update A/B tests [here](https://github.com/Automattic/wp-calypso/blob/master/test/e2e/config/default.json#L62)

File: config/default.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
 		"skipThemesSelectionModal",
 		"springSale30PercentOff",
 		"jetpackSignupGoogleTop",
-		"onlyJetpackMonthly",
+		"jetpackMontlyPlansOnly",
```
Comment (tyxla) [inline config/default.json]: Noting here as well that we have a typo here.

**Comment by johnHackworth** [comment]
d'oh! :man_facepalming: 

I missed that... thank you folks, I'll fix it :+1:

**Pull request closed without merging**